aws/ssm/parameters.go

```diff
@@ -0,0 +1,41 @@
+package ssm
+
+import (
+	"os"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ssm"
+	log "github.com/sirupsen/logrus"
+)
+
+// GetParameter takes in the variable key and checks OS for it
+// if there is no environment variable for the key, it will pull
+// from Parameter Store (Encrypted)
+// Variables in parameter store must all be encrypted
+func GetParameter(key string) (string, error) {
+	value := os.Getenv(key)
+	if value != "" {
+		return value, nil
+	}
+	withDecryption := true
+	sess, err := session.NewSessionWithOptions(session.Options{
+		Config:            aws.Config{Region: aws.String("us-west-2")},
+		SharedConfigState: session.SharedConfigEnable,
+	})
+
+	if err != nil {
+		log.Fatal("Error getting a new session:", err)
+		return "", err
+	}
+	ssmsvc := ssm.New(sess, aws.NewConfig().WithRegion("us-west-2"))
+	param, err := ssmsvc.GetParameter(&ssm.GetParameterInput{
+		Name:           &key,
+		WithDecryption: &withDecryption,
+	})
+	if err != nil {
+		log.Fatal("Error getting SSM parameter:", err)
+		return "", err
+	}
+	return *param.Parameter.Value, nil
```

Comment:
unsafe access to pointer, You may check if it not nil first

Reply:
It is already checking for error returned in previous statement & accessing pointer if no error.

Reply:
If you are sure it's fine